Classes/GHNSURL+Utils.h

```diff
@@ -129,7 +131,7 @@
  @param s String to escape
  @result Encode string
  */
-+ (NSString *)gh_encode:(NSString *)s;
++ (NSString *)gh_encode:(NSString *)s DEPRECATED_MSG_ATTRIBUTE("Use -[NSString stringByAddingPercentEncodingWithAllowedCharacters:] with the appropriate NSCharacterSet for each part of the URL. You can't encode a whole URL properly.");
```

Comment:
It looks like we're barely using this, want to just delete it instead of deprecating it?